README.md

```diff
@@ -170,7 +170,7 @@ If you like this project, please give it a star
 
 ## 🙏 Acknowledgments <a name="acknowledgements"></a>
 
```

Comment:
Great job team✋

- [ ] [Optional] Kindly ensure the GitHub handle link is directed to the correct GitHub Profile page. This will make your readme file very professional.
- [ ] Kindly fix the broken issues link to make your readme file more professional.